web-components/src/[sandbox]/examples/phone-input.ts

```diff
@@ -3,6 +3,16 @@ import { html } from "lit-element";
 
 export const phoneInputTemplate = html`
   <div>
+    <div class="row">
+      <md-phone-input
+        show-flags
+        errorMessage="This phone number is invalid"
+        @phoneinput-keydown="${(e: CustomEvent) => console.log("keydown: ", e.detail.value)}"
+        @phoneinput-blur="${(e: CustomEvent) => console.log("blur: ", e.detail.value, " is valid: ", e.detail.isValid)}"
+        @phoneinput-change="${(e: CustomEvent) =>
+          console.log("change: ", e.detail.value, " is valid: ", e.detail.isValid)}"
```

Comment:
Although I mentioned that there are console logs later that should be removed, these ones don't concern me due to them being in the sandbox.